packages/documentsearch-extension/schema/plugin.json

```
@@ -47,7 +47,14 @@
       "selector": ".jp-mod-searchable"
     }
   ],
-  "properties": {},
+  "properties": {
+    "searchDebounceTime": {
+      "title": "Search debounce time (ms)",
+      "description": "The debounce time in milliseconds applied to the search input field",
+      "type": "number",
+      "default": 500
```

Let's enforce [minimum value](https://json-schema.org/understanding-json-schema/reference/numeric.html#range):

```suggestion
      "default": 500,
      "exclusiveMinimum": 0
```

---

Done